background_scripts/main.js

```diff
@@ -295,6 +356,28 @@ const BackgroundCommands = {
   moveTabLeft: moveTab,
   moveTabRight: moveTab,
 
+  async setZoom({ tabId, registryEntry }) {
+    const zoomLevel = registryEntry.optionList[0] ?? 1;
+    const newZoom = parseFloat(zoomLevel);
+    // Chrome breaks on values below 0.01 and 10 or above.
+    if (!isNaN(newZoom) && newZoom >= 0.01 && newZoom < 10) {
+      chrome.tabs.setZoom(tabId, newZoom);
```

Comment:
Only try a zoom if we found a reasonable number.
We could also support parsing percents if you would like, so that users who try to enter:
`map z1 setZoom 10%`
will still have their config parse. That would not be difficult, but I think it's probably the right option might be to only have one correct method to set values.

Reply:
Agreed.